doc/source/advanced-use.rst

```diff
@@ -249,3 +249,30 @@ Now simply create a jail and supply the pkgs.txt file:
 ``iocell create pkglist=/path-to/pkgs.txt tag=myjail``
 
 This will install ``nginx`` and ``tmux`` in the newly created jail.
+
+Using volumes
+-------------
+
+Jails can have shared directories from the host which is implemented using nullfs(5). The idea is based on volumes used in the docker world. Files can also be Volumes and are copied over at jail startup.
+
+The ``volumes`` property has the following syntax:
```

Comment:
Consistency seems more important to me. It would be better to use `volume`, then switch all properties with the same issue to plural forms separately and at the same time.